sdk/storage/AzureStorageBlob/Source/Options/ClientMethodOptions.swift

```diff
@@ -208,34 +383,34 @@ public struct DeleteBlobOptions: RequestOptions {
     public let snapshot: Rfc1123Date?
 
     /// Request timeout in seconds.
-    public let timeoutInSeconds: Int?
-
-    /// Initialize a DeleteBlobOptions` structure.
-    /// - Parameters:
-    ///   - clientRequestId: A client-generated, opaque value with 1KB character limit that is recorded in analytics
-    ///     logs.
-    ///   - cancellationToken: A token used to make a best-effort attempt at canceling a request.
-    ///   - dispatchQueue: A dispatch queue on which to call the completion handler. Defaults to `DispatchQueue.main`.
-    ///   - deleteSnapshots: `DeleteBlobSnapshot` value to specify how snapshots should be handled.
-    ///   - timeoutInSeconds: Request timeout in seconds.
-    public init(
-        clientRequestId: String? = nil,
-        cancellationToken: CancellationToken? = nil,
-        dispatchQueue: DispatchQueue? = nil,
-        deleteSnapshots: DeleteBlobSnapshot? = nil,
-        snapshot: Date? = nil,
-        timeoutInSeconds: Int? = nil
-    ) {
-        self.clientRequestId = clientRequestId
-        self.cancellationToken = cancellationToken
-        self.dispatchQueue = dispatchQueue
-        self.deleteSnapshots = deleteSnapshots
-        self.snapshot = Rfc1123Date(snapshot)
-        self.timeoutInSeconds = timeoutInSeconds
-    }
+    public let timeout: TimeInterval?
+
+//    /// Initialize a DeleteBlobOptions` structure.
```

Comment:
Commented out?

Reply:
Removed. Most other options had these in place, my guess because of TransferManager, but delete operations do not need to (and cannot) be paused, so this would never be serialized.